transmart-api-server/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
 
     //Custom dependencies
     compile project(':transmart-rest-api')
+    //Deprecated plugin - user queries related functionality has been moved to an external application
```

Comment:
My comment is not related to this PR. Why we prefered to copy notification functionality to gb-backend/gb-backend instead of copying transmart-notifiction submodule to gb-backend